search-enterprise/latest/using/index.md

```diff
@@ -265,57 +148,109 @@ Prefixing any search qualifier with a `-` excludes all results that are matched
 
 When you search using negation the query will be built using the default field operator (OR). For instance, if you are querying for negation:
 
-```sql
+```afts
 taxi AND driver NOT yellow
 ```
 
 the query will be executed as:
 
-```sql
+```afts
 taxi AND driver OR NOT passenger
 ```
 
 If you expect a different behavior, you have to specify the boolean conjunction:
 
-```sql
+```afts
 taxi AND driver AND NOT passenger
 ```
 
-### Search for optional, mandatory, and excluded elements of a query
+## Search for optional, mandatory, and excluded elements of a query
 
-Sometimes AND and OR are not enough. If you want to find documents that must contain the term "car", score those with the term "red" higher, but do not match those just containing "red".
+Sometimes `AND` and `OR` are not enough. If you want to find documents that must contain the term "car", score those with the term "red" higher, but do not match those just containing "red".
 
 |Operator|Description|
 |--------|-----------|
 |","|The field, phrase, group is optional; a match increases the score.|
-|"+"|The field, phrase, group is mandatory (Note: this differs from Google - see "=")|
+|"+"|The field, phrase, group is mandatory **Note:** this differs from Google - see "=")| @engineering should there an an = here too?
```

Comment:
Missing info that should be requested from @engineering

Reply:
has had tech review by Angel